data/the emerald sword.txt

```diff
@@ -0,0 +1,803 @@
+			`	"You can call it Dread. Regret. Loss. You're standing in the graveyard of a lost civilization."`
+			label who
+			`	"I am the guardian of this site. I am also the one who guided you towards the buried city, so your species would bear witness and take heed of the lessons of ones who have long succumbed to their own folly. I did not intend for you to reach this far."`
+			`	"Here is where the Sheragi came to an end. The tools that drove them to extinction are long gone. We witnessed... as they cast those tools into the heat of this star.`
+			choice
+				`	"You mean this ship? Its weapons systems seem to be intact."`
+					goto cannon
+				`	"What tools?"`
+			`	"Ones capable of crippling an entire world's worth of life."`
+				goto next
+			label cannon
+			`	"That ship is just a relic. The weapon that perished the Sheragi was indiscriminate. An irrational decision they made themselves."`
+			label next
+			`	"The feeling in your chest becomes overpowering and you hold on to the command seat, gasping for air, light-headed. It seems evident those feelings are not your own, but are washing through you like a tidal wave. As you struggle to regain composure, the thoughts continue to form in your mind.`
+			`	"I know what you seek here. The trinket you covet is of little relevance, even for your own, human standards. But if you will it, then take it. Perhaps the sight of that vessel will remind the feeble living races of those who have come and gone... and how, despite their great accomplishments, as they turn to war they are reduced to graveyards, relics, and memories"`
+			`	"The voice, and the overwhelming emotions that came alongside it, disappear as mysteriously as they came. Desyo, who witnessed you speaking into thin air, asks whether you feel okay. It seems like no one else heard it."`
+
+	npc board
+		government "Sheragi"
+		personality derelict mute
+		system Zubenelhakrabi
+		ship "Emerald Sword" "Emerald Sword"
+		conversation
+			`You dock with the Emerald Sword, and the crew starts working to carefully find a way in, directed by the Hai archaeologists. There is no atmosphere inside it, as the life support has long since ceased to work, and it takes a long time to restore it through your ship's life support and patch tiny micrometeorite holes. Once it's in breathable conditions, it still smells like a tomb. You find many Sheragi skeletons and mummified remains. A few of them have prosthetics that survived better than their host. Your crew gets to work relocating the remains to the cargo bay, and cleaning up the thick layer of dust and grime that covers every surface, while the more specialized crew tries to bring the power core back to life, and take advantage of the zero-G environment to move the jump drive into the ship more easily. You'll need to land to properly install it, however.`
+	
+	on complete
+		#outfit "Broken Ship Systems" -1
+		#outfit "Refurbished Ship Systems" 1
```

Comment:
Wouldn't it work to have just the removal of "Broken Systems" on landing in a separate mission? That mission should require the "Broken Systems" outfit to offer and just take it away.
Re-adding "Refurbished systems" doesn't seem necessary and setting up a temporary outfitter also doesn't seem necessary in such case.

Reply:
That is commented out because it's what i tried when i first made that mission. Apparently, the game doesn't check escorts for outfits to be removed and reinstalled, only the flagship and cargo. I was originally going to make the mission slightly longer, first making the ship flyable, but with a constant debuff, and then to remove that debuff, retrofitting it's internal space for a human crew.